packages/teleport/src/services/database/databases.test.ts

```diff
@@ -58,19 +58,17 @@ describe('correct formatting of all type and protocol combos', () => {
   );
 });
 
-const mockResponse = {
-  items: [
-    {
-      name: 'aurora',
-      desc: 'PostgreSQL 11.6: AWS Aurora',
-      protocol: 'postgres',
-      type: 'rds',
-      uri:
-        'postgres-aurora-instance-1.c1xpjrob56xs.us-west-1.rds.amazonaws.com:5432',
-      labels: [
-        { name: 'cluster', value: 'root' },
-        { name: 'env', value: 'aws' },
-      ],
-    },
-  ],
-};
+const mockResponse = [
```

Comment:
Also add a test, if response comes back undefined. Arrays in particular may come as undefined.

Reply:
If you want to know why: https://github.com/golang/go/issues/37711